sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -2736,6 +2736,12 @@ case class ParquetTableReader(
   }
 
   override def close(): Unit = {
+    debugDumpPrefix.foreach { prefix =>
```

Comment:
nit: this feels like it is a bug fix that is unrelated to the C2C code change. Could we do it in a separate PR?

Reply:
Done, removed.